README.md

```diff
@@ -97,15 +107,43 @@ client = Client(url='https://conjur.myorg.com',
                 ca_bundle='/path/to/my/ca/bundle.pem')
 ```
 
-#### With `.netrc` and `.conjurrc` settings
+#### With `.netrc` and `.conjurrc` files
 
-Write the code same as in the first example but create the client with the following arguments:
+You can provide the above parameters passed in as part of the Client initialization process in two files 
+(`.netrc` and `.conjurrc`). Both `.netrc` and `.conjurrc` files should be saved to your home directory. 
+For example `~/.netrc`. 
 
-```python3
+The `.conjurrc` contains connection details needed to connect to the Conjur endpoint and should 
+contain details for cert_file, conjur_account, and conjur_url.
+
+```
+# .conjurrc
+---
+cert_file: /Users/someuser/conjur-server.pem
+conjur_account: someaccount
+conjur_url: https://conjur-server
+```
+
+The `.netrc` file or (`_netrc` for Windows environments) contains credentials needed to login the 
+Conjur endpoint and should consist of machine, login, and password. Note that the value for 
+'machine' should have `/authn` appended to the end of the url and 'password' should contain the 
+API key of the user you intend to log in as.
```

Comment:
@sigalsax I think this should be reviewed also by @sashaCher or @moticless

Reply:
@InbalZilberman what do we need them to review here? writing this or providing this option?

Reply:
@InbalZilberman in terms of what do we want to check?
@moticless and I spoke about this and we dont have a problem with writing this as long as we tell the user to follow least privilege and ensures that no other user can access this file